daemon/api.go

```diff
@@ -327,6 +332,22 @@ func getSnapsInfo(c *Command, r *http.Request) Response {
 		}
 	}
 
+	if len(includeTypes) > 0 {
+		for i, r := range results {
+			hasType := false
+			for _, t := range includeTypes {
+				if r["type"] == t {
+					hasType = true
+					break
+				}
+			}
+
+			if !hasType {
+				delete(results, i)
```

Comment:
(nitpick) I got a bit confused by the "i" here, I assumed its a list or slice. But results is a map, so I wonder if it might be easier to use "k" or "key" or "name" for the variable "i" so that its easier to see that i not a integer (in my mental model most "i" are integers :)